pods/server/src/__start.ts

```diff
@@ -83,7 +83,9 @@ if (frontUrl === undefined) {
 }
 
 const sesUrl = process.env.SES_URL
+const cursorMaxTime = process.env.SERVER_CURSOR_MAXTIMEMS
```

Comment:
Please set default value here, if not passed, since you missed a bunch of docker-compose.yaml files.

Reply:
You have it in deep. ok.